toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -33,6 +34,39 @@ while getopts ":c:u:" OPTIONS; do
   esac
 done
 
+# If we are running unattended, we need to make sure that we only run once.
+# We can use the unattended config file via flock to synchronize the unattended
+# install flows.
+if [[ ! -f "$UNATTENDED_CONFIG_FILE" ]]; then
```

Comment:
shouldn't the `!` be removed?

Reply:
Huh... will need to dig into this to see why this is working  at all then...

Reply:
Ah, I see the issue. `$ISO_ROOT` isn't mounted yet so this always returns true. This logic needs to move down after we complete the mount process.

(Unrelated, but this race condition illuminates why the mount functionality may have started failing earlier)

Reply:
Could you explain a bit what would have caused the mount failure? I think I've seen it once or twice before when trying ISO install on both devices. Not sure if this change also help fix it.

Reply:
The old retry code would loop until the `mount` command returned code 0. But... the mount command will return non-zero when the mount point is already mounted. This would mean that if the mount was already mounted it would get stuck in the retry loop until it timed out. This likely started being an issue when we added the parallel installers since only one would be able to successfully call the mount command, the other would end up stuck in the retry loop. I added the grep to escape early if it was already mounted but I couldn't for the life of me figure out what was causing the mount to already be present. In hindsight... it was the other installer running on ttyS0.